package.json

```diff
@@ -25,6 +25,7 @@
     "@types/tmp": "^0.2.0",
     "@typescript-eslint/eslint-plugin": "^4.2.0",
     "@typescript-eslint/parser": "^4.2.0",
+    "date-fns": "^2.16.1",
```

Comment:
Seems like this is used "in production" (and should be a dep), no?

Reply:
It's just used in tests for the date comparison, so no. It's just a dev dep.

Reply:
Ah. Gotcha, sorry thought I saw it outside tests too.